### PR TITLE
tools/profile: add multi process/thread support

### PR DESCRIPTION
**Pull request by yezhem**
Just like the perf tool, support multi process or thread may be useful.

### CODE REVIEW

**Comment by yonghong-song** [comment]
Do you actually have a use case for this?

**Comment by yezhem** [comment]
> 

Yes, but in a productive environment, can`t paste the use case example.

**Comment by yezhem** [comment]
> Do you actually have a use case for this?

@yonghong-song Yes, but in a productive environment, can`t paste the use case example.

**Comment by yonghong-song** [comment]
> > Do you actually have a use case for this?
> 
> @yonghong-song Yes, but in a productive environment, can`t paste the use case example.

okay. thanks for clarification.

File: tools/profile_example.txt [hunk 1]
```diff
@@ -161,6 +161,35 @@ but we're lacking the symbol translation. This is a common for all profilers
 on Linux, and is usually fixable. See the DEBUGGING section of the profile(8)
 man page.
 
+You can also profile different process:
+# ./profile -p 2040 1316151
```
Comment (yonghong-song) [inline tools/profile_example.txt:165]: should the option look like `-p 2040 -p 1316151`? What is the syntax `perf` allows multiple pid/tid's?

Comment (yezhem) [reply tools/profile_example.txt:165]: perf just like: -p 2040,1316151 
should change to it?

Comment (yonghong-song) [reply tools/profile_example.txt:165]: Yes, it would be good to following perf conventions.

Comment (yonghong-song) [reply tools/profile_example.txt:165]: Please also mention perf convention in your commit message.

Comment (yezhem) [reply tools/profile_example.txt]: resolved, pls review it

File: tools/profile.py [hunk 2]
```diff
@@ -81,9 +88,9 @@ def stack_id_err(stack_id):
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
 thread_group = parser.add_mutually_exclusive_group()
-thread_group.add_argument("-p", "--pid", type=positive_int,
+thread_group.add_argument("-p", "--pid", type=positive_int_list,
     help="profile process with this PID only")
```
Comment (yonghong-song) [inline tools/profile.py:92]: could you change this help to 'profile process with one or more comma separated PIDs only'

File: tools/profile.py [hunk 3]
```diff
@@ -81,9 +88,9 @@ def stack_id_err(stack_id):
...
-thread_group.add_argument("-L", "--tid", type=positive_int,
+thread_group.add_argument("-L", "--tid", type=positive_int_list,
     help="profile thread with this TID only")
```
Comment (yonghong-song) [inline tools/profile.py:94]: change this to 'prrofile thread with one or more comman separated TIDs only'.

**Review by yonghong-song — COMMENTED** [review]
LGTM. Thanks. A couple of minor comments. Could you also update man page for '-p' and '-L' option?

File: tools/profile.py (hunk 3 above)
Comment (yezhem) [reply tools/profile.py]: @yonghong-song Thanks for your review, pls check the modification.

**Pull request merged**